Agenda/Controllers/History/MonthDetailsViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  MonthDetailsViewController.swift
+//  Agenda
+//
+//  Created by Егор Бадмаев on 03.04.2022.
+//
+
+import UIKit
+
+final class MonthDetailsViewController: UIViewController {
+    
+    private let month: Month
+    private let coreDataManager: CoreDataManagerProtocol
+    
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(AgendaTableViewCell.self, forCellReuseIdentifier: AgendaTableViewCell.identifier)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.rightBarButtonItem = editButtonItem
+        title = month.date?.formatToMonthYear()
+        view.backgroundColor = .white
+        
+        setupViewAndConstraints()
+    }
+    
+    init(month: Month, coreDataManager: CoreDataManagerProtocol) {
+        self.month = month
+        self.coreDataManager = coreDataManager
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupViewAndConstraints() {
+        view.addSubview(tableView)
+        
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
+        ])
```

Comment:
It's better to fix trouble when title scrolls with table view and we see different background colors

Reply:
I've added UIView with height equals 1 point. That helped me